src/test/scala/com/fulcrumgenomics/sv/AlignedSegmentTest.scala

```diff
@@ -18,58 +18,63 @@ class AlignedSegmentTest extends UnitSpec {
   "AlignedSegment" should "be built from a SamRecord" in {
     implicit val builder: SamBuilder = new SamBuilder()
 
+    def test(rec: SamRecord, expected: AlignedSegment): Unit = {
+      val actual = AlignedSegment(rec)
+      actual shouldBe expected.copy(recs=IndexedSeq(rec))
+    }
+
```

Comment:
Lots of changes here, but all are just to accommodate having the `SamRecord` tracked in an `AlignedSegment`.   No new tests.